ehr_sqlite3.py

```diff
@@ -0,0 +1,188 @@
+"""SQLite3 Assignment."""
```

Comment:
Before I add the test cases and documentation 
can you please review my work and check if I am on the right track